self-hosted-agent/helm/variables.tf

```diff
@@ -0,0 +1,22 @@
+variable "cluster" {
```

Comment:
maybe change this to `cluster_name` ? there are a lot of AWS identifiers (ARN, name, id, ...) so I think being more specific here would be good